mscore/plugin/api/elements.h

```diff
@@ -223,6 +223,11 @@ class Element : public Ms::PluginAPI::ScoreElement {
       API_PROPERTY( veloChangeMethod,        VELO_CHANGE_METHOD        )
       API_PROPERTY( veloChangeSpeed,         VELO_CHANGE_SPEED         )
       API_PROPERTY( dynamicRange,            DYNAMIC_RANGE             )
+      /**
+       *    The way a ramp interpolates between values.
+       *    \since MuseScore 3.5
+       */
+      API_PROPERTY( changeMethod,            CHANGE_METHOD             )
```

Comment:
I guess `ChangeMethod` enum should also be exposed to plugin API then. Also `Pid::CHANGE_METHOD` could be assigned to the corresponding hairpin property (in `get/setProperty` functions) to make it possible to make use of this unified property in plugins. But this can be done separately of this pull request, we lack proper support for spanners in plugins anyway.